docs/build.sh

```diff
@@ -135,7 +137,8 @@ generate_api_rst v3
 # Fixup anchors and references in v3 so they form a distinct namespace.
 # TODO(htuch): Do this in protodoc generation in the future.
 find "${GENERATED_RST_DIR}"/api-v3 -name "*.rst" -print0 | xargs -0 sed -i -e "s#envoy_api_#envoy_v3_api_#g"
-find "${GENERATED_RST_DIR}"/api-v3 -name "*.rst" -print0 | xargs -0 sed -i -e "s#config_resource_monitors#v3_config_resource_monitors#g"
+# TODO(phlax): Remove this once above is removed
+find "${GENERATED_RST_DIR}"/api-v3 -name "*.rst" -print0 | xargs -0 sed -i -e "s#envoy_v2_api_#envoy_api_#g"
```

Comment:
my motivation to remove this mangling altogether is highlighted quite well here - ie having to workaround the workaround